### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by marlowesphone** [redeclipse/base#1420] [Help and Support] [answered]
Where can I download the source code of red eclipse? I would like to make a modified version of the game

**Comment by qreeves** [comment, marked as answer]
You can clone the source code with GIT or by downloading a ZIP from the main page for the `base` repository:
![image](https://user-images.githubusercontent.com/3297929/229398852-b1a97028-4f71-4e4b-8fca-a36adaeb7383.png)

The download link for the ZIP at: https://github.com/redeclipse/base/archive/refs/heads/master.zip

**Reply by qreeves** [reply]
I should note that the data for the game (maps, sounds, textures, etc) are contained in different repositories. While the source code is [ZLIB licensed](https://github.com/redeclipse/base/raw/master/doc/license.txt), the game data has varying licenses. Also note that the name/logo/etc for "Red Eclipse" is subject to our [Trademark Policy](https://github.com/redeclipse/base/raw/master/doc/trademark.txt).

See the list of repositories on our [org page](https://github.com/redeclipse) for the [data modules](https://github.com/redeclipse/base/tree/master/data).
